homeworks/andrii.tkachenko_Hodr1337/2-logic/logic.js

```diff
@@ -0,0 +1,25 @@
+const firstNumber = 20;
+const secondNumber = 22;
+const operation = '*';
+switch (operation) {
+    case '*':
+        console.log(firstNumber * secondNumber);
+        break;
+    case '/':
+        console.log(firstNumber / secondNumber);
+        break;
+    case '+':
+        console.log(firstNumber + secondNumber);
+        break;
+    case '-':
+        console.log(firstNumber - secondNumber);
+        break;
+    case '**':
+        console.log(firstNumber ** secondNumber);
+        break;
+    case '%':
+        console.log(firstNumber % secondNumber);
```

Comment:
It's okay, but let's make it through an additional variable called result. We need to decrease console.log in code. It should be one instead of six.